app/app.go~HEAD

```diff
@@ -0,0 +1,245 @@
+package app
```

Comment:
the file shouldn't be called `app.go~HEAD` but `app.go`

Reply:
https://github.com/cosmos/peggy/pull/63/commits/1e4c1e7bd6febbb1bc68144caed1529a3e2754f1